docs/node/mainnet/e-upgrade.md

```diff
@@ -100,3 +100,310 @@ upgrade. For now, please take a look at the related [Testnet upgrade].
 :::
 
 [Testnet upgrade]: ../testnet/upgrade-log.md
+
+### Voting
+
+On XXX, an upgrade proposal will be proposed which (if accepted) will schedule
+the upgrade on epoch **XXX**.
+
+For an optimized voting experience, we recommend utilizing the [Oasis CLI].
+Follow these steps to cast your vote:
+1. [Import your keys into the wallet]
```

Comment:
We should make sure to update the link once the new CLI release which will include easier file import. But this is probably fine for now.